racket/src/thread/thread.rkt

```diff
@@ -479,3 +492,95 @@
       (set-thread-ignore-break-cells! t (cond
                                           [(eq? ignore bc) #f]
                                           [else (hash-remove ignore bc)])))))
+;; ----------------------------------------
+;; Thread mailboxes
+(define (enqueue-mail! thd v)
```

Comment:
Can you annotate which functions are called in atomic mode (see other comments in this file)?